spec/workflow_service_spec.rb

```diff
@@ -128,7 +128,7 @@
     end
 
     it 'should return false if the PUT to the DOR workflow service throws an exception' do
-      expect{ Dor::WorkflowService.update_workflow_status(@repo, @druid, 'errorWF', 'reader-approval', 'completed') }.to raise_error(Exception, /the server responded with status 400/)
+      expect{ Dor::WorkflowService.update_workflow_status(@repo, @druid, 'errorWF', 'reader-approval', 'completed') }.to raise_error(Dor::WorkflowException, /status 400/)
     end
```

Comment:
It is, indeed, unfortunate that we have long lines in this code base. I'd rather not introduce code style changes along side these functional changes, though. The rubocop file in this project sets the line length to 300, and we can happily revisit that in the future.